src/components/overview/addToCart/AddToCart.jsx

```diff
@@ -12,46 +12,75 @@ const AddToCart = (props) => {
 
   const [selectedSize, setSize] = useState('')
   const [selectedSKU, setSKU] = useState('')
+  const [selectedQuantity, setQuantity] = useState(1);
 
   //TODO
   //Based on the selected size, get SKU, creat quantity selector
   //If no sizes, selector only has "OUT of Stock"
   //If size not selected, render "-" for quantity selector
 
   const handleSizeSelect = (event) => {
-    var skuID = event.target.value[0]
-    var skuData = event.target.value[1]
-
+    var skuID = event.target.value
+    var skuData = currentSkus[skuID]
     setSKU(skuID)
     setSize(skuData.size)
   }
 
-  //TODO: Needs test
-
-  //Style Selector
   let styleSelector;
-  if (Object.entries(currentSkus).length > 0){
 
-    styleSelector =
-             ( <select value={selectedSize} onChange={handleSizeSelect}>
+  if (Object.entries(currentSkus).length > 0){
+    styleSelector = (
+             <select name={selectedSize} onChange={handleSizeSelect}>
 
                 {selectedSize === '' && <option>Select Style</option>}
```

Comment:
Should the Size selector say 'Select Size'? Right now it's labeled select style

Reply:
Thanks for catching that. Pushed up the changes.